Scripts/header.sh

```
@@ -0,0 +1,98 @@
+
+# Define the header template
+header_template="//
+//  %s
+//  %s
+//
+//  Created by %s.
+//  Copyright © %s %s.
+//
+//  Permission is hereby granted, free of charge, to any person
+//  obtaining a copy of this software and associated documentation
+//  files (the “Software”), to deal in the Software without
```

Shellcheck (0.11.0)

[warning] This is a unicode quote. Delete and retype it (or ignore/singlequote for literal).

(SC1111)